src/components/PageHeader.tsx

```diff
@@ -59,14 +60,24 @@ export const PageHeader = ({ title }: Props) => {
           </a>
           <div className={classes.title}>{title}</div>
         </div>
-        <a
-          className={classes.contact}
-          href={process.env.REACT_APP_FIDER_LINK}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Contact
-        </a>
+        <div>
+          <a
+            className={classes.contact}
+            href="/fireWeatherCalculator"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            FireWeather
+          </a>
```

Comment:
Do you think we need this @Sybrand ? Cause my understanding was each product(pages) are gonna be independent

Reply:
We explicitly agreed that they were going to be independent, so agreed - I don't know where this came from.

Reply:
@Sybrand @Kyubinhan yes fire weather is a independent component. I just added this link to as as a temporary solution. This will be removed when actual routing would be added to the application. There is already another ticket for this purpose wps #196.

Reply:
@Sybrand @Kyubinhan I removed the link.